concepticondata/conceptlists/Gruehn-2008-200.tsv

```diff
@@ -0,0 +1,201 @@
+Gruehn-2008-200-61	61	ernst	serious			50.072	4.583	4.083	5.25	5.333	4.333	5.292	4.813	4.667	5.333	4.417	4.667	5	4.542	4.771	5.417	5.167	4.333	5	5.292	4.667	4.979	4.583	4.917	5.25	4	4.75	4.625	4.688	4.833	4.75	4.667	4.917	4.792	4.792	4.792	4.083	3.917	3.833	4.083	4	3.958	3.979	4.625	4.625	1.917
+Gruehn-2008-200-62	62	erschoepft	exhausted			10.781	3.917	3.75	2.833	2.833	3.833	2.833	3.333	3.75	2.75	3.417	2.833	3.25	3.125	3.188	2.5	2	3.273	2.75	2.25	3	2.617	4.583	5.167	5.083	5.75	4.875	5.417	5.146	4	3.25	2.667	2.75	3.625	2.708	3.167	4.25	4	4.083	3.667	4.125	3.875	4	2.625	3.25	2
+Gruehn-2008-200-63	63	erstaunt	astonished			10.944	4.75	4.583	4.5	4.083	4.667	4.292	4.479	4	4.417	4	4.75	4.208	4.375	4.292	3.25	2.667	3.25	3.5	2.958	3.375	3.167	5.083	6	4.667	5.25	5.542	4.958	5.25	3.917	3.417	4.5	3.5	3.667	4	3.833	3.25	3.167	2.5	2.583	3.208	2.542	2.875	4.813	4.75	1.917
+Gruehn-2008-200-64	64	feige	cowardly	3812	COWARDLY	2.852	1.833	1.833	2.083	1.917	1.833	2	1.917	5.167	5.5	4.667	4.083	5.333	4.375	4.854	2.417	1.667	2.917	3.167	2.042	3.042	2.542	3.667	4.417	4.417	4.5	4.042	4.458	4.25	2.917	2.333	2	2.083	2.625	2.042	2.333	2.75	2.667	2.583	3	2.708	2.792	2.75	1.75	2.125	1.458
+Gruehn-2008-200-65	65	feindselig	hostile			1.011	1.333	1.75	1.417	1.5	1.542	1.458	1.5	6.5	6.167	6.333	6.417	6.333	6.375	6.354	3.083	4.25	4.75	3.333	3.667	4.042	3.854	3.833	4.667	4.583	5.333	4.25	4.958	4.604	1.583	2.5	1.333	1.5	2.042	1.417	1.729	2.833	3.333	3.083	2.75	3.083	2.917	3	1.688	1.188	1.083
+Gruehn-2008-200-66	66	fies	nasty			632	1.167	1.75	1.25	1.417	1.458	1.333	1.396	5.917	5.5	6.417	5.25	5.708	5.833	5.771	4.5	4.273	4	3.417	4.391	3.708	4.043	4.75	5.917	4.583	5.5	5.333	5.042	5.188	2.25	3.25	1.583	1.583	2.75	1.583	2.167	2.583	2.417	2.917	3.167	2.5	3.042	2.771	1.688	1.188	1
+Gruehn-2008-200-67	67	flexibel	flexible	1691	FLEXIBLE	08.04	5.75	5	6.083	5.333	5.375	5.708	5.542	2.75	3.083	3.25	3.917	2.917	3.583	3.25	6.083	5.583	4.333	5.417	5.833	4.875	5.354	2.727	2.583	4.083	3.5	2.652	3.792	3.234	5.5	5.5	6.167	5.667	5.5	5.917	5.708	1.75	1.417	2.25	2.75	1.583	2.5	2.042	5.938	5.875	1.667
+Gruehn-2008-200-68	68	frech	impudent			4.976	3.917	3.583	2.333	2	3.75	2.167	2.958	4.083	4.583	5.083	5.083	4.333	5.083	4.708	5.083	4.417	5.083	3.333	4.75	4.208	4.479	5.083	6.333	5.333	5.75	5.708	5.542	5.625	4.667	4.75	3	2.583	4.708	2.792	3.75	1.917	1.833	1.833	1.917	1.875	1.875	1.875	4.75	4.375	1.125
+Gruehn-2008-200-69	69	friedlich	peaceful			12.432	6.583	6.083	5.833	5.833	6.333	5.833	6.083	1.833	1.833	1.75	2.667	1.833	2.208	2.021	5	4.833	3.917	4.417	4.917	4.167	4.542	4.75	5.333	4.75	5.25	5.042	5	5.021	6.167	6.083	5.917	5.667	6.125	5.792	5.958	3.667	4.25	3.833	4	3.958	3.917	3.938	5.938	5.938	1.333
+Gruehn-2008-200-70	70	froehlich	cheerful	1976	MERRY	11.241	6.583	6.417	6.417	6.25	6.5	6.333	6.417	2	2.25	2	2.455	2.125	2.217	2.17	5.727	4.5	5.25	4.833	5.087	5.042	5.064	6.083	6.083	6.25	6.417	6.083	6.333	6.208	5.917	5.583	5.25	5	5.75	5.125	5.438	2.417	2.5	2.667	2.917	2.458	2.792	2.625	6.688	6.5	1.708
```

Comment:
Our existing mappings are not consistent with respect to this concept, might be worthwhile to look at that. The mapping is fine here I'd say.

Reply:
@MiraAhmedovic Could you open an issue so that we can discuss the consistency of the mappings later.